Userland/mount.cpp

```diff
@@ -96,11 +96,20 @@ bool mount_all()
             continue;
         }
 
-        dbg() << "Mounting " << devname << "(" << fstype << ")"
+        const char* filename = parts[0].characters();
+        int fd = open(filename, O_RDONLY);
```

Comment:
Will the kernel like readonly descriptors though? Wouldn't it return an error when you try to write a block?